README.md

```diff
@@ -41,7 +41,8 @@ cd llvm-project/llvm
 mkdir build
 cd build
 cmake .. -DLLVM_ENABLE_PROJECTS="clang;openmp" \ # openmp is optional
-  -DCMAKE_INSTALL_PREFIX=${LLVM_INSTALL_DIR}
+  -DCMAKE_INSTALL_PREFIX=${LLVM_INSTALL_DIR} \
+  -DLLVM_TARGETS_TO_BUILD=X86 # Also optional, but if "All" is selected, potential issues can arise. Furthermore, speeds up compilation time if limited to just the architectures you actually use. Possible values: AArch64, AMDGPU, ARM, AVR, BPF, Hexagon, Lanai, Mips, MSP430, NVPTX, PowerPC, RISCV, Sparc, SystemZ, VE, WebAssembly, X86, XCore
```

Comment:
You probably meant that users build the LLVM with a host target.
```suggestion
  -DLLVM_TARGETS_TO_BUILD=host # Also optional, but if "All" is selected, potential issues can arise. Furthermore, speeds up compilation time if limited to just the architectures you actually use.
```
What also should speed up the LLVM build is to the `-DCMAKE_BUILD_TYPE=Release` option. The `Debug` option is default build type for LLVM 14.